cts/valgrind-pcmk.suppressions

```diff
@@ -6,14 +6,6 @@
    fun:crm_get_msec
 }
 
-{
-   Ignore option parsing
```

Comment:
I am seeing leaks in `crm_log_preinit()`/`crm_log_init()` -> `qb_log_filter_ctl2()`, and various ->  `qb_log_callsite_get()`, which seem more likely to be in libqb than pacemaker. The callsite one could be valgrind now treating some of our debug/trace calls as lost rather than reachable, but it seems odd that it's just some of them.

But then I'm seeing this one that's definitely Pacemaker:

    14 bytes in 2 blocks are definitely lost in loss record 6 of 62
        at 0x484482F: malloc (vg_replace_malloc.c:446)
        by 0x4AE812E: strdup (in /usr/lib64/libc.so.6)
        by 0x49A283E: pcmk__str_copy_as (strings.c:1252)
        by 0x48B79F6: pcmk__new_cancel_action (pcmk_sched_recurring.c:661)
        by 0x48B2808: UnknownInlinedFun (pcmk_sched_recurring.c:281)
        by 0x48B2808: UnknownInlinedFun (pcmk_sched_recurring.c:626)
        by 0x48B2808: pcmk__primitive_create_actions (pcmk_sched_primitive.c:848)
        ...

I don't see how that's possible, because anytime we allocate an action, we add it to the scheduler object. It wouldn't be possible to leak just an action (not to mention just an action name, which this is) without leaking the rest of the object.

Reply:
> The callsite one

Does it say "definitely lost"? In limited testing, I only ran into ones that said "possibly lost". That can happen if strange things are done with indexing. And oh boy, are there. Look at how `qb_list_for_each_safe()`/`qb_list_entry()` gets a log filter to free it, for example.

Reply:
No, it's all "possibly". It hasn't shown these in the past though, so maybe its detection changed.